### PR TITLE
duplicity 2.1.4

### PR DESCRIPTION
**Pull request by branchv**
Created with `brew bump-formula-pr`.

### CODE REVIEW

File: Formula/d/duplicity.rb [hunk 1]
```diff
@@ -412,11 +405,9 @@ class Duplicity < Formula
...
   end
 
-  # Using GitHub tarball as requirements.txt is missing from PyPI tarball.
-  # Issue ref: https://github.com/dropbox/stone/issues/266
   resource "stone" do
-    url "https://github.com/dropbox/stone/archive/refs/tags/v3.3.1.tar.gz"
-    sha256 "dc5aff3fad1333188d4ddb4eee0a19d31e6262bb3cdf10c0bbdaeb309ff91a52"
+    url "https://files.pythonhosted.org/packages/99/6f/ef25bbc1aefeb9c905d527f1d3cd3f41f22f40566d33001b8bb14ae0cdaf/stone-3.3.1.tar.gz"
+    sha256 "4ef0397512f609757975f7ec09b35639d72ba7e3e17ce4ddf399578346b4cb50"
   end
```
Comment (chenrui333) [inline Formula/d/duplicity.rb:411]: this might need to be reverted

File: Formula/d/duplicity.rb [hunk 2]
```diff
@@ -35,18 +35,6 @@ class Duplicity < Formula
...
-      url "https://files.pythonhosted.org/packages/d6/f4/154cf374c2daf2020e05c3c6a03c91348d59b23c5366e968feb198306fdf/jeepney-0.8.0.tar.gz"
-      sha256 "5efe48d255973902f6badc3ce55e2aa6c5c3b3bc642059ef3a91247bcfcc5806"
-    end
-
-    resource "secretstorage" do
-      url "https://files.pythonhosted.org/packages/53/a4/f48c9d79cb507ed1373477dbceaba7401fd8a23af63b837fa61f1dcd3691/SecretStorage-3.3.3.tar.gz"
-      sha256 "2403533ef369eca6d2ba81718576c5e0f564d5cca1b58f73a8b23e7d4eeebd77"
-    end
-  end
-
```
Comment (chenrui333) [inline Formula/d/duplicity.rb:49]: this might be still needed

File: Formula/d/duplicity.rb (hunk 1 above)
Comment (branchvincent) [reply Formula/d/duplicity.rb]: yup fixed

File: Formula/d/duplicity.rb (hunk 2 above)
Comment (branchvincent) [reply Formula/d/duplicity.rb]: fixed

**Comment by branchvincent** [comment]
Updated the extra resources from https://gitlab.com/duplicity/duplicity/-/blob/rel.2.1.4/requirements.txt, which added `atom`, `botocore`, `httplib2` and dropped `boto`

**Review by p-linnane — APPROVED** [review]

**Pull request merged**